File: Python/Step2/2753.py
year = int(input())

if year % 4 == 0 and year % 100 != 0 or year % 400 == 0:
    print(1)
else:
    print(0)

"""
과정 및 함수 설명
함수

과정
1. 년도를 입력받는다.
2. 윤년 여부를 출력한다.
"""

File: C/백준/Step2/2753.c
#include <stdio.h>

int main () {
    int year;
    int m1, m2, m3;

    scanf("%d", &year);

    m1 = year % 4;
    m1 = year % 100;
    m1 = year % 400;

    if ((m1 == 0 && m2 != 0) || m3 == 0) {
        printf("1");
    } else {
        printf("0");
    }

    return 0;
}